docs/user_guide/make_fit.md

```diff
@@ -1,30 +1,47 @@
 <!--
 (C) copyright 2019 University Centre in Svalbard (UNIS)
 author: Emma Bland, UNIS
+
+Modifications: 2022-06 Emma Bland (UNIS) Added fitting algorithm descriptions and updated examples
 -->
 # RAWACF to FITACF
 
-SuperDARN data is distributed to the community in the `rawacf`  format. For most science applications you will want to process these files into the `fitacf` format using the `make_fit` routine.
+SuperDARN data is distributed to the community in the `rawacf` format. For most science applications you will want to process these files into the `fitacf` format using the `make_fit` routine.
+
+## Choosing a fitting algorithm
+
+Several methods have been developed by the SuperDARN community to determine the physical properties of SuperDARN backscatter (power, velocity, spectral width, elevation, and their errors).
+
+The fitting algorithms available for processing SuperDARN data are:
+
+| Name       | Notes |
+| :----------| :-----------|
+| `fitacf3`  | Suitable for all scientific applications except for convection mapping due to additional data at midlatitudes affecting the Heppner-Maynard boundary determination. Fitted data may appear 'noisy' for some radars due to operational problems (remove with [`fit_speck_removal`](despecking.md)).<br>**Released:** 2017 <br>**References:** (1) [FitACF 3.0 White Paper](https://superdarn.github.io/dawg/files/sup_material/FITACF3_white_paper.pdf), (2) [SuperDARN noise estimation](https://doi.org/10.1002/essoar.10510616.1)|
+| `fitacf2`  | Suitable for all scientific applications, including convection mapping. Generally results in fewer fitted ACFs compared to `fitacf3`. <br>**Released:** 2006<br>**Reference:** [Annales Geophysicae, 24, 115–128, 2006](https://doi.org/10.5194/angeo-24-115-2006) |
```

Comment:
> @pasha-ponomarenko the mid-latitude echoes are affecting FITACF2 as badly as they affect FITACF3...This problem was essentially ignored until we stumbled across it while testing FITACF3, which created a wrong impression that the HMB issue is attributed solely to FITACF3. I believe that the Readthedocs text should be corrected respectively...

I wrote it that way because that was the PIs' conclusion about the HMB in December 2021. But your presentation makes it clear what's going on, so I'm optimistic that we can transition to the spectral width method for determining the HMB for the next RST release anyway. I've updated the text with the suggestion above...is that better?